code/modules/hydroponics/trays/tray.dm

```diff
@@ -124,6 +124,17 @@
 		)
 
 /obj/machinery/portable_atmospherics/hydroponics/AltClick()
+	if (istype(usr, /mob/living/carbon/alien/diona))//A diona alt+clicking feeds the plant
```

Comment:
Need to stop reading code late at night.

Though, if you're going to be using such a check more often (which you probably are, idno), it might be an idea to make a proc/isnymph(). Similar to how we have ishuman() and whatever else. (One might already exist, as well.) It'll make the code less abstract.